mongo/errors.go

```diff
@@ -539,3 +572,29 @@ func processWriteError(err error) (returnResult, error) {
 		return rrAll, nil
 	}
 }
+
+// joinBatchErrors appends messages from the given errors to a comma-separated string. If the
+// target length is reached, it stops appending error messages and appends the message
+// "+N more errors..." to the end.
+//
+// Example format:
+//     "[message 1, message 2, +8 more errors...]"
+func joinBatchErrors(errs []error, targetLength int) string {
```

Comment:
Instead of accepting a `targetLength`, can this always use `batchErrorsTargetLength`? Both callers always specify 2000.

Reply:
Yeah, that makes sense. Will update.